docs/spec.openapi.yaml

```diff
@@ -0,0 +1,425 @@
+openapi: 3.0.3
+
+info:
+  title: Proxy Python
+  license:
+    name: MIT
+    url: https://github.com/hawks-atlanta/proxy-python/blob/main/LICENSE
+  version: 1.0.0
+
+tags:
+  - name: Authentication
+  - name: Account
+  - name: Authorization
+  - name: Files
+
+paths:
+  /login:
+    post:
+      tags:
+        - Authentication
+      description: Authenticates to the server
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/credentials'
+      responses:
+        '201':
```

Comment:
`201` stands for `created`. Since we're not creating or storing any resource in this request, I think we should use the `200` code.